<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectDefinitionDialog</class>
 <widget class="QDialog" name="ProjectDefinitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project definition</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="namePushButton">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QPushButton" name="tagPushButton">
       <property name="text">
        <string>Tag:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="3">
      <widget class="QLineEdit" name="tagLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="authorPushButton">
       <property name="text">
        <string>Author:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="authorLineEdit">
       <property name="minimumSize">
        <size>
         <width>340</width>
         <height>0</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Dates, from:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="2">
      <widget class="QDateEdit" name="startDateEdit"/>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>to:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QDateEdit" name="finishDateEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="descriptionPushButton">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QPushButton" name="crsPushButton">
       <property name="text">
        <string>CRS:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="3">
      <widget class="QLineEdit" name="crsLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="outputPathPushButton">
       <property name="text">
        <string>Output path:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="6">
      <widget class="QLineEdit" name="outputPathLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>628</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
